apps/openmw/mwworld/projectilemanager.hpp

```diff
@@ -3,6 +3,7 @@
 
 #include <string>
 
+#include <osg/observer_ptr>
```

Comment:
> /usr/include/c++/v1/utility:253:9: error: implicit instantiation of undefined template 'osg::observer_ptr<osg::Camera>'

`osg::observer_ptr<osg::Camera>` is used in components/sceneutil/lightmanager.hpp, so the include should be added there.

https://github.com/OpenMW/openmw/commit/aec8c3846103da0d65ca772d859c501f2d6d9c57